### PR TITLE
Completed css-four-column-layout challenge.

### PR DESCRIPTION
**Pull request by sharon-tieu**
Screen Shot 2022-08-29 at 4 55 32 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.container {
+  margin: 50px 100px;
+}
+
+.col-full {
+  width: 100%;
+}
+
+.col-one-fourth {
+  width: calc(100% /6);
```
Comment (BrettAlbright) [inline css-four-column-layout/styles.css:38]: make sure that your column widths match the semantic name they are given! for one fourth it should be `100%/4`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great Sharon! Just make sure to update the column width!

**Comment by sharon-tieu** [comment]
Screen Shot 2022-08-29 at 4 55 32 PM

**Comment by sharon-tieu** [comment]
Screen Shot 2022-08-29 at 5 08 42 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**